components/support/payload/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "payload-support"
```

Comment:
totally unrelated to this patch, but something I've noticed in the past is that the names of our packages aren't great for being in a repo like m-c. I wonder if we should start using a longer `package` name, but add a `lib` section where the name used by the code is specified.

(This sounds fine until I try and think of what a good longer name would be, and I come up with something like `app-services-payload-support`, then realize that's just as bad - if not more - in a moz-central context.

Anyway, leaving this comment in even though I don't think it's worth doing anything about :)